src/views/transactions/components/account-selector/account-selector.view.jsx

```diff
@@ -25,7 +25,11 @@ function AccountSelector({
 
   React.useEffect(() => {
     if (accountsTask.status === "pending" && poolTransactionsTask.status === "successful") {
-      onLoadAccounts(undefined, poolTransactionsTask.data, fiatExchangeRates, preferredCurrency);
+      if (transactionType === TxType.Deposit) {
+        onLoadAccounts(fiatExchangeRates, preferredCurrency);
```

Comment:
If we are loading the accounts without the `poolTransactionsTask` why put it inside the parent `if`? I'll put it outside and then we can remove the `poolTransactionsTask` from the reducer and its actions I think.

Reply:
Done